www/hanp05/ukol7/main.js

```diff
@@ -0,0 +1,84 @@
+//deklarace proměnných 
+var card1 = null;
+var card2 = null;
+var score = 0;
+var cardsFlipped = 0;
+
+var cars = ['Škoda', 'Audi', 'BMW', 'Citroen', 'Chrysler', 'Chevrolet', 'Ford', 'Kia', 'Hyundai', 'Ferrari'];
+cars = cars.concat(cars);
+cars.sort(function () {
+    return 0.5 - Math.random();
+});
+
+var gameArea = document.getElementById('gameArea');
+var scoreText = document.getElementById('score');
+
+var assignCards = function (card) {
+    if (!card1) { //pokud ještě není přiřazena první karta, tak ji přiřaď
+        card1 = card;
+        return false;
+    }
+    card2 = card;// jinak přiřaď druhou
+};
+
+var resetCardProperties = function () {
+    setTimeout(function () {
+        card1.classList.remove('flipped');
+        card2.classList.remove('flipped');
+        card1 = null;
+        card2 = null;
+    }, 500);
```

Comment:
500ms je docela kratka doba na zapamatovani :))), to by slo z nej udelat parametr pro obtiznost hry co myslis? :)